students/mariusz_michalczyk/lesson_05_lists/maximum.py

```diff
@@ -0,0 +1,22 @@
+def get_array_based_on_user_input():
```

Comment:
Why not `array_from_input` (God kills a kitten for every preposition, "get", "generic" and "abstract" :)